File: leet-code/561. Array Partition/561_array_partition.py
def array_pair_sum(nums) -> int:
    # 정렬 후 짝수 인덱스 값만 슬라이싱한 리스트 총합 리턴
    return sum(sorted(nums)[::2])


print(array_pair_sum([1, 4, 3, 2]))
print(array_pair_sum([6, 2, 6, 5, 1, 2]))

File: leet-code/561. Array Partition/561_array_partition.dart
class Solution {
  int arrayPairSum(List<int> nums) {
    nums.sort();
    int sums = 0;

    for (int i = 0; i < nums.length; i++) {
      if (i % 2 == 0) {
        sums += nums[i];
      }
    }

    return sums;
  }
}

void main() {
  Solution solution = Solution();
  print(solution.arrayPairSum([1, 4, 3, 2]));
}
